src/components/contact.js

```diff
@@ -1,52 +1,95 @@
-import React from 'react';
+import React, {Component} from 'react';
+import EmailForm from './emailForm';
 
-export default function Contact (props){
-  const sectionStyle = {
-    textAlign: 'center',
-    paddingTop: '30%',
-    // fontSize: '20px',
-  }
-  const nameStyle = {
-    color: '#282828',
-    fontSize: '35px'
-  }
-  const profStyle = {
-    color:'#282828',
-    fontSize: '20px',
-    padding: '3%'
-  }
+ class Contact extends Component{
+   constructor (props){
+     super (props);
+   }
 
-  const websiteLinkStyles = {
-    fontSize: '2.5em',
-    color: '#282828',
-    paddingLeft: '5%'
-  }
 
-  const emailLogoStyles = {
-    fontSize: '2.5em',
-    color: '#282828',
-    // paddingLeft: '5%'
-  }
+  renderAlert(){
+    const errorStyle = {
```

Comment:
In React, styles are defined in the JS file only if one of the attributes is dynamic. Here I can see that most, if not all, of the styles are static. When the styles are static it is better to put them in a CSS class (within a CSS file, such that there is one CSS file for each kind of component- and then call these CSS files from the HTML file) and call that class in the corresponding DOM elements.